src/engraving/rw/read114/read114.cpp

```diff
@@ -2000,14 +2000,41 @@ static void readMeasure(Measure* m, int staffIdx, XmlReader& e, ReadContext& ctx
                     j->setPlayUntil(e.readText());
                 } else if (t == "continueAt") {
                     j->setContinueAt(e.readText());
-                } else if (t == "playRepeats") {
-                    j->setPlayRepeats(e.readBool());
                 } else if (t == "subtype") {
-                    e.readInt();
+                    e.skipCurrentElement(); // obsolete, always "Repeat"
                 } else if (!TRead::readTextProperties(j, e, ctx)) {
                     e.unknown();
                 }
             }
+
+            // infer jump type
+            QString jumpTo = j->jumpTo();
+            QString playUntil = j->playUntil();
+            QString continueAt = j->continueAt();
+            if (jumpTo == "start") {
+                if (playUntil == "end") {
+                    j->setJumpType(JumpType::DC);
+                } else if (playUntil == "fine") {
+                    j->setJumpType(JumpType::DC_AL_FINE);
+                } else if (playUntil == "coda" && continueAt == "codab") {
```

Comment:
Is this too strict?
Would relaxing or even dropping that check (makeing it the `else` branch) also recognize more complex jumps like jumps to a 1st and a 2nd Coda?